server/controllers/article.controller.js

```diff
@@ -20,25 +20,123 @@ class ArticleController {
     */
     if (req.file) req.body.image = req.file.secure_url;
     const {
-      title, body, tags, description, image, published
+      title, body, tagsList, description, published
     } = req.body;
     const { id } = req.user;
     const successMessage = 'Article successfully created';
     const errorMessage = 'Article was not created successfully';
+    const tags = [...new Set(tagsList)];
```

Comment:
Good job. I like the initiative you took in the aspect of filtering the tags. For the sake of consistency throughout the project, and to stay in line with sequelize conventions, I suggest you name your models starting with a capital letter.  So 'tag' should be 'Tag' and articletag should be 'Articletag'.

Reply:
Ok thank you, would follow the model conventions on going